### PR TITLE
[TRTLLM] T5 python backend log_probs support

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description

To support log_probs in TensorRT-LLM in details for T5.

### CODE REVIEW

File: engines/python/setup/djl_python/tensorrt_llm.py [hunk 1]
```diff
@@ -29,6 +30,68 @@
 from djl_python.properties_manager.properties import is_rolling_batch_enabled
 
 
+def _get_value_based_on_tensor(value, index=None):
```
Comment (lanking520) [inline engines/python/setup/djl_python/tensorrt_llm.py:33]: could we fuse all python backend interaction into a single class (file)? Just trying to keep the handler clean

Comment (sindhuvahinis) [reply engines/python/setup/djl_python/tensorrt_llm.py:33]: Sure, we could do that. Same as stable diffusion handlers.

Comment (sindhuvahinis) [reply engines/python/setup/djl_python/tensorrt_llm.py:33]: Sure, we could do that. Same as stable diffusion handlers.

File: engines/python/setup/djl_python/utils.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import logging
+from djl_python.inputs import Input
+from djl_python.encode_decode import encode, decode
+from djl_python.chat_completions.chat_utils import is_chat_completions_request, parse_chat_completions_request
+
+
+def parse_input(
```
Comment (lanking520) [inline engines/python/setup/djl_python/utils.py:7]: Could we reuse this across handlers?

**Review by lanking520 — APPROVED** [review]

**Pull request merged**